### ISSUE TITLE
Remove unused `alt-hero` field from expunge-assist.md project file

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#3196]
### Overview
As a developer, I want to make sure unused code is removed from the codebase in project cards so that
- time to load the website is as low as possible,
- it's easier to understand the code, and
- time to review the code is reduced.

### Details
In project md files, `image-hero` is a decorative background image and `alt-hero` is the alt text associated with `image-hero`. However, `alt-hero` text is not used anywhere in our codebase since the `image-hero` is used as an SCSS background image.

Moreover, if the `image-hero` were to be used in an img HTML tag in the future, the alt text for all the `image-hero` images would be `alt=""`, which could be set programmatically for all the alt properties associated with each `image-hero`.   

### Action Items
- [x] For the file `_projects/expunge-assist.md`, remove line 9 for the alt-hero
```
alt-hero: 'Expunge Assist project logo'
```
- [x] Once the pull request associated with this issue is approved and merged, please update and edit epic #2722 by
  - [x] Checking off the dependency for this issue
  - [x] If all dependencies are checked off, please move issue #2722 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- This issue is part of epic #2722 

#### File and code links you will need to work on this issue
- Directory to find the page once you have it in your IDE: `_projects/expunge-assist.md`
- Click on the link below to see the source file code:
https://github.com/hackforla/website/blob/fdecdb49ef86a3cc3bf4617effe055e39efecc13/_projects/expunge-assist.md?plain=1#L9

#### Information about WCAG
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)

**Comment by rolzar** [comment]
Availability: 5 hrs.
ETA: 10/7.

**Comment by bishrfaisal** [comment]
Hey @rolzar, we see that you have taken up your second issue. Awesome work so far! It has been 2 weeks since you self-assigned, have you run into any obstacles? If so, please let us know and we would be happy to help!

**Comment by bishrfaisal** [comment]
Hi @rolzar! We haven't heard from you since October 4th (19 days ago). Hope you are okay. We have un-assigned you from this issue. Please feel free to pickup a new issue from the prioritized backlog when you return!

**Comment by roslynwythe** [comment]
Availability: 10/27, 10/28 daytime, 10/29 evening, 10/30
ETA: 10/30

**Comment by roslynwythe** [comment]
I'm still proceeding down the pre-work checklist and just learned that not supposed to assign a "good second issue" until my first issue PR has been merged, so therefore I am unassigning myself.

**Comment by roslynwythe** [comment]
Availability: daytime 10/28, all day 10/30 and 10/31
ETA: 10/31
